src/opm/output/eclipse/LoadRestart.cpp

```diff
@@ -1195,7 +1195,7 @@ namespace {
                 // proofing and robustness in case that ever changes.
                 const auto groupOrderIx = (gname == "FIELD")
                     ? groupData.maxGroups() // NGMAXZ -- Item 3 of WELLDIMS
-                    : std::max(group.seqIndex(), std::size_t{1}) - 1;
+                    : std::max(group.insert_index(), std::size_t{1}) - 1;
```

Comment:
Please update the comment above to ensure that it does not refer to `seqIndex()` anymore.

Reply:
ok